README.md

```diff
@@ -1,16 +1,51 @@
-# Release Integration Tests
+# Release Integration Test
 
+This README provides instructions on running a test using the provided shell script.
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## Test Steps and Expected Result
+
+1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+   The test is running against RHTAP-Stage, so a user wants to monitior the test running throught UI or CLI 
+   will need an access to the cluster and be joined to the workspaces *release-dev-team* and *release-managed-team*. 
+   
+   Our test expect to eun on both workspaces above, and it's expected (based on the fllow agreed on), namespaces dev and managed exist, ploicy, service account, secrets already deployed on those two dedicated workspaces 
+   *release-dev-team* and *release-managed-team*.
+
+   Required that a user to run the test when connecting the RHTAP if the user is not logged in to SSO, so a browser page will pop-up and require an SSO login. 
+
+   Before running the test script, make sure you have the following in place:
+   - **Kubeconfig Files:** Ensure that you have valid kubeconfig files for both the `DEV_KUBECONFIG` and `MANAGED_KUBECONFIG` paths provided in the script.
+
+2. **Running the Test:**
+    The test script sets up resources, waits for pipeline runs to complete, checks release status, and reports the result.
+
+3. **Expected Result:**
+
+   a. `We exppect that a Build PipelineRun in dev workspace cstarted and Succeed.`
+
+   b. `A Release CR is created in dev workspace.`
+
+   c. `Release PipelineRun is started in managed workspace, fbc-release pipeline and we expect it Succeed.`
+   
+   d. `The Release CR expecetd to to be updated when (c) pass, and CR is Released, if Failed will report Fail.`
```

Comment:
Lots of typos in this one. Just going to rewrite it as a suggestion
The Release CR will be updated with details of the Release PipelineRun once it passes. 
e. The Release CR will be marked as Released in its status

We are running a static test. It shouldn't be "if fail." We should know if it will fail or not. These are expected results. I'd think we expect it to pass, so no need to mention fail stuff